ci-operator/step-registry/ipi/conf/azure/ipi-conf-azure-ref.yaml

```diff
@@ -27,9 +27,13 @@ ref:
       in some regions or accounts due to quota issues.
 
   - name: COMPUTE_NODE_TYPE
-    default: 'Standard_D4s_v3'
```

Comment:
Why not set `Standard_D4ps_v5` in azure-arm64 test?

Reply:
Hi @vrutkovs, I'd say this is compatible with the previous work done on aws: for arm64 (#32969) and in general (#31684) to let the CI use default instance types for cost-saving purposes, unless a specific test needs to override the COMPUTE_NODE_TYPE.

Reply:
ipi-conf-azure is not just for arm64 test,  setting it to "" in ref.yaml and giving default value for both amd64 and arm64 in shell file is more easy.